packages/devtools_app/lib/src/screens/memory/panes/leaks/primitives/analysis_status.dart

```diff
@@ -18,11 +18,18 @@ enum AnalysisStatus {
   ShowingError,
 }
 
-Duration _showingResultDelay = const Duration(seconds: 5);
-Duration _delayForUiToHandleState = const Duration(milliseconds: 5);
+class _Constants {
```

Comment:
Constants don't need to be defined in an enclosing class.

https://dart.dev/guides/language/effective-dart/design#avoid-defining-a-class-that-contains-only-static-members